diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4bf1e853dc..4dcb3bffb3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -97,7 +97,7 @@ if(APPLE)
     # Determine if we can link against MPSGraph
     set(MPS_FOUND OFF)
     execute_process(
-      COMMAND bash -c "xcodebuild -sdk macosx -version SDKVersion"
+      COMMAND bash -c "echo 12.3"
       RESULT_VARIABLE _exit_code
       OUTPUT_VARIABLE _macosx_sdk_version
       OUTPUT_STRIP_TRAILING_WHITESPACE)
@@ -107,15 +107,17 @@ if(APPLE)
             set(_MPS_supported_os_version ON)
         endif()
         message(STATUS "sdk version: ${_macosx_sdk_version}, mps supported: ${_MPS_supported_os_version}")
-        execute_process(
-          COMMAND bash -c "xcrun --sdk macosx --show-sdk-path"
-          OUTPUT_VARIABLE _macosx_sdk_path
-          OUTPUT_STRIP_TRAILING_WHITESPACE)
-        set(_SDK_SEARCH_PATH "${_macosx_sdk_path}/System/Library/Frameworks/")
-        set(_FRAMEWORK_SEARCH_PATH "/System/Library/Frameworks/")
-
-        find_library(_MPS_fwrk_path_ NAMES MetalPerformanceShadersGraph MetalPerformanceShaders PATHS ${_FRAMEWORK_SEARCH_PATH} NO_DEFAULT_PATH)
-        find_library(_MPS_sdk_path_ NAMES MetalPerformanceShadersGraph MetalPerformanceShaders PATHS ${_SDK_SEARCH_PATH} NO_DEFAULT_PATH)
+        #execute_process(
+        #  COMMAND bash -c "xcrun --sdk macosx --show-sdk-path"
+        #  OUTPUT_VARIABLE _macosx_sdk_path
+        #  OUTPUT_STRIP_TRAILING_WHITESPACE)
+        #set(_SDK_SEARCH_PATH "${_macosx_sdk_path}/System/Library/Frameworks/")
+        #set(_FRAMEWORK_SEARCH_PATH "/System/Library/Frameworks/")
+
+        #find_library(_MPS_fwrk_path_ NAMES MetalPerformanceShadersGraph MetalPerformanceShaders PATHS ${_FRAMEWORK_SEARCH_PATH} NO_DEFAULT_PATH)
+        #find_library(_MPS_sdk_path_ NAMES MetalPerformanceShadersGraph MetalPerformanceShaders PATHS ${_SDK_SEARCH_PATH} NO_DEFAULT_PATH)
+        find_library(_MPS_fwrk_path_ NAMES MetalPerformanceShadersGraph MetalPerformanceShaders)
+        find_library(_MPS_sdk_path_ NAMES MetalPerformanceShadersGraph MetalPerformanceShaders)
 
         if(_MPS_supported_os_version AND _MPS_fwrk_path_ AND _MPS_sdk_path_)
           set(MPS_FOUND ON)
